xml/Microsoft.Extensions.Logging.Console/ConsoleFormatter.xml

```diff
@@ -91,12 +91,12 @@
         <Parameter Name="textWriter" Type="System.IO.TextWriter" />
       </Parameters>
       <Docs>
-        <typeparam name="TState">To be added.</typeparam>
-        <param name="logEntry">To be added.</param>
-        <param name="scopeProvider">To be added.</param>
-        <param name="textWriter">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <typeparam name="TState">The type of the object to be written.</typeparam>
+        <param name="logEntry">The log entry.</param>
+        <param name="scopeProvider">The provider of scope data.</param>
+        <param name="textWriter">The string writer embedding ansi code for colors.</param>
+        <summary>Writes the log message to the specified TextWriter.</summary>
+        <remarks>if the formatter wants to write colors to the console, it can do so by embedding ANSI color codes into the string.</remarks>
```

Comment:
```suggestion
        <remarks>If the formatter wants to write colors to the console, it can do so by embedding ANSI color codes into the string.</remarks>
```